tensorflow/compiler/tf2xla/kernels/strided_slice_op.cc

```diff
@@ -99,6 +97,30 @@ class StridedSliceOp : public XlaOpKernel {
       slice = ctx->builder()->Rev(slice, dimensions_to_reverse);
     }
 
+    if (!simple_strides) {
```

Comment:
It took me a while to work out what's going on here --- perhaps add a brief comment explaining the general approach?